packages/terra-demographics-banner/src/DemographicsBanner.jsx

```diff
@@ -3,100 +3,154 @@ import PropTypes from 'prop-types';
 import { injectIntl, intlShape } from 'react-intl';
 import DemographicsBannerDisplay from './DemographicsBannerDisplay';
 
+const identifierShape = {
+  /**
+   * The label of the identifier: i.e.: `SSN`
```

Comment:
Nit: 
```suggestion
   * The label of the identifier: e.g.: `SSN`
```
And same below